frontend/src/lib/lemon-ui/ProfilePicture/ProfilePicture.tsx

```diff
@@ -16,6 +17,7 @@ export interface ProfilePictureProps {
     title?: string
     index?: number
     isSystem?: boolean
+    isBot?: boolean
```

Comment:
This makes it possible for a user to both be "system" and "bot", that seems confusing

Reply:
Maybe `type?: 'person' | 'bot' | 'system'`?

Reply:
:chefs-kiss: